fuzz/src/full_stack.rs

```diff
@@ -14,918 +14,1220 @@
-use bitcoin::hashes::sha256::Hash as Sha256;
-use bitcoin::hash_types::{Txid, BlockHash, WPubkeyHash};
 
 use lightning::chain;
-use lightning::chain::{Confirm, Listen};
 use lightning::chain::chaininterface::{BroadcasterInterface, ConfirmationTarget, FeeEstimator};
 use lightning::chain::chainmonitor;
-use lightning::chain::transaction::OutPoint;
 use lightning::chain::keysinterface::{InMemorySigner, KeysInterface};
-use lightning::ln::channelmanager::{BestBlock, ChainParameters, ChannelManager, PaymentHash, PaymentPreimage, PaymentSecret};
-use lightning::ln::peer_handler::{MessageHandler,PeerManager,SocketDescriptor};
+use lightning::chain::transaction::OutPoint;
+use lightning::chain::{Confirm, Listen};
+use lightning::ln::channelmanager::{BestBlock, ChainParameters, ChannelManager};
 use lightning::ln::msgs::DecodeError;
-use lightning::routing::router::get_route;
+use lightning::ln::peer_handler::{MessageHandler, PeerManager, SocketDescriptor};
+use lightning::ln::{PaymentHash, PaymentPreimage, PaymentSecret};
 use lightning::routing::network_graph::NetGraphMsgHandler;
+use lightning::routing::router::get_route;
 use lightning::util::config::UserConfig;
-use lightning::util::events::{EventsProvider,Event};
 use lightning::util::enforcing_trait_impls::EnforcingSigner;
+use lightning::util::events::{Event, EventsProvider};
 use lightning::util::logger::Logger;
 use lightning::util::ser::Readable;
 
 use utils::test_logger;
 use utils::test_persister::TestPersister;
 
-use bitcoin::secp256k1::key::{PublicKey,SecretKey};
+use bitcoin::secp256k1::key::{PublicKey, SecretKey};
 use bitcoin::secp256k1::Secp256k1;
 
 use std::cell::RefCell;
-use std::collections::{HashMap, hash_map};
 use std::cmp;
+use std::collections::{hash_map, HashMap};
+use std::sync::atomic::{AtomicU64, AtomicUsize, Ordering};
 use std::sync::{Arc, Mutex};
-use std::sync::atomic::{AtomicU64,AtomicUsize,Ordering};
 
 #[inline]
 pub fn slice_to_be16(v: &[u8]) -> u16 {
-	((v[0] as u16) << 8*1) |
-	((v[1] as u16) << 8*0)
+    ((v[0] as u16) << 8 * 1) | ((v[1] as u16) << 8 * 0)
 }
 
 #[inline]
 pub fn slice_to_be24(v: &[u8]) -> u32 {
-	((v[0] as u32) << 8*2) |
-	((v[1] as u32) << 8*1) |
-	((v[2] as u32) << 8*0)
+    ((v[0] as u32) << 8 * 2) | ((v[1] as u32) << 8 * 1) | ((v[2] as u32) << 8 * 0)
 }
 
 #[inline]
 pub fn slice_to_be32(v: &[u8]) -> u32 {
-	((v[0] as u32) << 8*3) |
-	((v[1] as u32) << 8*2) |
-	((v[2] as u32) << 8*1) |
-	((v[3] as u32) << 8*0)
+    ((v[0] as u32) << 8 * 3)
+        | ((v[1] as u32) << 8 * 2)
+        | ((v[2] as u32) << 8 * 1)
+        | ((v[3] as u32) << 8 * 0)
 }
 
 #[inline]
 pub fn be64_to_array(u: u64) -> [u8; 8] {
-	let mut v = [0; 8];
-	v[0] = ((u >> 8*7) & 0xff) as u8;
-	v[1] = ((u >> 8*6) & 0xff) as u8;
-	v[2] = ((u >> 8*5) & 0xff) as u8;
-	v[3] = ((u >> 8*4) & 0xff) as u8;
-	v[4] = ((u >> 8*3) & 0xff) as u8;
-	v[5] = ((u >> 8*2) & 0xff) as u8;
-	v[6] = ((u >> 8*1) & 0xff) as u8;
-	v[7] = ((u >> 8*0) & 0xff) as u8;
-	v
+    let mut v = [0; 8];
```

Comment:
Formatting changes in `full_stack.rs` look undesirable. See commit 5a478c769dc106ea0a0a0747973ef2d0f67288fe.

Reply:
Looks like rustfmt accidentally got run across this file?